samples/cpp/classification_sample_async/README.md

```diff
@@ -14,36 +14,41 @@ The following C++ API is used in the application:
 | Infer Request Operations | `ov::InferRequest::get_input_tensor` | Get an input tensor. |
 | Tensor Operations | `ov::shape_size`, `ov::Tensor::data` | Get a tensor shape size and its data. |
 
-Basic OpenVINO™ Runtime API is covered by [Hello Classification C++ sample](../hello_classification/README.md).
+Basic OpenVINO™ Runtime API is covered by [Hello Classification C++](../hello_classification/README.md).
 
 | Options | Values |
 | :--- | :--- |
 | Validated Models | [alexnet](@ref omz_models_model_alexnet), [googlenet-v1](@ref omz_models_model_googlenet_v1) |
-| Model Format | OpenVINO™ toolkit Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
+| Model Format | OpenVINO Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
 | Supported devices | [All](../../../docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md) |
 | Other language realization | [Python](../../../samples/python/classification_sample_async/README.md) |
 
 ## How It Works
 
-At startup, the sample application reads command line parameters and loads the specified model and input images (or a
-folder with images) to the OpenVINO™ Runtime plugin. The batch size of the model is set according to the number of read images. The batch mode is an independent attribute on the asynchronous mode. Asynchronous mode works efficiently with any batch size.
+In the beginning, the sample application reads command-line parameters and loads the specified model and input images (or a
+folder with images) to the OpenVINO Runtime plugin. The batch size of the model is set according to the number of read images. The batch mode is an independent attribute on the asynchronous mode. Asynchronous mode works efficiently with any batch size.
 
-Then, the sample creates an inference request object and assigns completion callback for it. In scope of the completion callback
+Next, the sample creates an inference request object and assigns completion callback for it. In scope of the completion callback,
 handling the inference request is executed again.
 
-After that, the application starts inference for the first infer request and waits of 10th inference request execution being completed. The asynchronous mode might increase the throughput of the pictures.
+After that, the application starts inference for the first infer request and waits until the 10th inference request execution is completed. The asynchronous mode might increase the throughput of the pictures
 
-When inference is done, the application outputs data to the standard output stream. You can place labels in .labels file near the model to get pretty output.
+When inference is done, the application outputs data to the standard output stream. Labels may placed in `.labels` file near the model to get pretty output.
```

Comment:
Grammer error: 
Labels may be placed in the '.labels' file near the model to get a prettier output. 

Is pretty necessary here?

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0

As for "pretty" - We can change it to something like:
"Labels may be placed in the '.labels' file near the model to get a better looking output." 
or "Labels may be placed in the '.labels' file near the model to make an output look cleaner".

Reply:
"Labels may be placed in the '.labels' file near the model to make an output look cleaner".

This sounds a lot better. I would change it to this.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/ee9ed62f53be4cfb2fa894f36dd3742765bf7dec